javascript/builtins/Array.json

```diff
@@ -1758,6 +1758,48 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "stable": {
+            "__compat": {
+              "description": "Stable",
```

Comment:
I think this should be "stable sorting" to avoid confusion with other uses of stable like, stable feature.

Reply:
That makes sense.